pkg/scheduler/ugm/manager_test.go

```diff
@@ -725,6 +725,24 @@ func TestDecreaseTrackedResourceForGroupTracker(t *testing.T) {
 	assert.Equal(t, groupTracker != nil, true)
 	assert.Equal(t, groupTracker.queueTracker.childQueueTrackers["parent"].runningApplications[TestApp1], false)
 	assert.Equal(t, resources.Equals(groupTracker.queueTracker.childQueueTrackers["parent"].resourceUsage, resources.Zero), true)
+
+	manager.IncreaseTrackedResource("root.parent", TestApp1, usage, user)
+	groupTracker = m.GetGroupTracker(user.Groups[0])
+	assert.Equal(t, groupTracker != nil, true)
+	assert.Equal(t, groupTracker.queueTracker.childQueueTrackers["parent"].runningApplications[TestApp1], true)
+	assert.Equal(t, resources.Equals(groupTracker.queueTracker.childQueueTrackers["parent"].resourceUsage, usage), true)
+
+	conf.Queues[0].Queues[0].Limits = []configs.Limit{
+		{
+			Limit:  "parent queue limit for a specific group",
+			Groups: user.Groups,
+		},
+	}
```

Comment:
How about just cleanup the setting of queue Limits?

```suggestion
	conf.Queues[0].Queues[0].Limits = nil
```

Besides, could we add comments for the two scenario while calling DecreaseTrackedResource()?

Reply:
done.